features/support/formatter_output_helpers.ts

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash'
+import { doesHaveValue, valueOrDefault } from '../../src/value_checker'
+import {
+  IJsonFeature,
+  IJsonScenario,
+  IJsonStep,
+} from '../../src/formatter/json_formatter'
+
+// Converting windows stack trace to posix and removing cwd
+//    C:\\project\\path\\features\\support/code.js
+//      becomes
+//    features/support/code.js
+function normalizeExceptionAndUri(exception: string, cwd: string): string {
+  return exception
+    .replace(cwd, '')
+    .replace(/\\/g, '/')
+    .replace('/features', 'features')
+}
+
+function normalizeProtobufObject(obj: any, cwd: string): void {
+  if (doesHaveValue(obj.uri)) {
+    obj.uri = normalizeExceptionAndUri(obj.uri, cwd)
+  }
+  if (doesHaveValue(obj.testResult)) {
+    if (doesHaveValue(obj.testResult.duration)) {
+      obj.testResult.duration.nanos = 0
+    }
+    if (doesHaveValue(obj.testResult.message)) {
+      obj.testResult.message = normalizeExceptionAndUri(
+        obj.testResult.message,
+        cwd
+      )
+    }
+  }
+}
+
+export function normalizeMessageOutput(
```

Comment:
Can typedocs be added to exported functions

Reply:
Same as above, will we dealt with in post merge tasks